crates/rooch-integration-test-runner/tests/cases/private_generics_indirect_call.move

```diff
@@ -1,29 +1,57 @@
 //# init --addresses creator=0x42
 
+//# publish
+module creator::test0 {
+    struct KeyStruct has key {
+        x: u64,
+    }
+}
+
 //# publish
 module creator::test {
-    use moveos_std::storage_context::{StorageContext};
-    use moveos_std::account_storage;
     struct Foo has key {
         x: u64,
     }
+    use std::string::{Self};
 
-    #[private(T1)]
-    public fun publish_foo<T1>(ctx: &mut StorageContext, s: &signer) {
-        account_storage::global_move_to<Foo>(ctx, s, Foo { x: 500 })
+    #[private_generics(T1)]
+    public fun publish_foo<T1>(s: &signer) {
+        move_to<Foo>(s, Foo { x: 500 })
```

Comment:
Do not use `move_to`. The Move global instructions will be deprecated in #248

Reply:
Yes, the `move_to` has been replaced with `account_storage::global_move_to`.